### DISCUSSION TITLE
Welcome to odftoolkit Discussions!

### DISCUSSION THREAD

**Discussion by svanteschubert** [tdf/odftoolkit#245] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Asbjoedt** [comment]
Hi!

We are working on a project which aim to create a specification for how to archive OpenDocument Spreadsheets and make an open-source validator available for the spec.

The draft v1.0 of the spec is now available: https://github.com/openpreserve/sheets-preservation-spec
The validator, which will not use the odftoolkit, is in-development: https://github.com/openpreserve/odf-validator

The project partners are Open Preservation Foundation, Danish National Archives, the National Archives of the Netherlands and the Estonian National Archives.

I have experimented with implementing the odftoolkit to validate the official ODF standard and the file format policies defined in our draft spec. I am not complete with this yet, and right now I am taking a break this work. Here's the repo: https://github.com/Asbjoedt/ODS-Archiving

We are very happy the odftoolkit exists and look forward to seeing it develop.

**Reply by svanteschubert** [reply]
@Asbjoedt 
I always love to see that ODF is being used and software is being written. 💯 

We might even consider to think about merging the [Odf Validator from the Open Preservation Foundation
](https://github.com/openpreserve/odf-validator) with the [Odf Validator from The Document Foundation
](https://github.com/tdf/odftoolkit/tree/master/validator).

Were there certain requirments that were not fulfilled? In any case, we should give it a try! 👍
